react/next/components/Thumbnails.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { memo, FC } from 'react'
+
+interface Props {
+  currentSlide: number
+  controls: string
+  classNames?: {
+    thumbnails?: string
+    thumbnail?: string
+    selectedThumbnail?: string
+  }
+  thumbnails?: {
+    items: Array<{
+      url: string
+      forSlide: number
+    }>
+    width: string
+  }
+  goToSlide: (index: number) => void
+}
+
+/**
+ * Slider Tumbnails
+ */
+const Thumbnails: FC<Props> = props => {
+  const { goToSlide, thumbnails, currentSlide, controls, classNames } = props
+
+  const getThumbClass = (index: number) =>
+    `${classNames!.thumbnail} ${
```

Comment:
Maybe using classnames would make this snippet more readable

Reply:
I'm avoiding the use of other libs in this implementation :)

Reply:
Maybe a way out would be to have a className implementation in utils